src/amo/components/GetFirefoxButton/index.js

```diff
@@ -41,7 +44,10 @@ type InternalProps = {|
 export const GetFirefoxButtonBase = (props: InternalProps) => {
   const { addon, buttonType, className, i18n, userAgentInfo } = props;
 
-  if (isFirefox({ userAgentInfo })) {
+  if (
+    buttonType === GET_FIREFOX_BUTTON_TYPE_NONE ||
```

Comment:
There may be times we only want to display a `GetFirefoxButton` on its own, not within an `InstallButtonWrapper`, which is why I wanted to control this here. Also, the logic for dealing with the other `buttonType`s is also in `GetFirefoxButton`, so it seems like keeping it all centralized is a good idea, rather than spreading it around components. Does that make sense?

Reply:
hmm I guess I thought this was specific to using this with the `InstallButtonWrapper` so (IMO) it made more sense to just use it there (if there was no `getFireFoxButtonType` prop passed - then it wouldn't be added). I was thinking that outside of this `InstallButtonWrapper`, if you didn't want to show the `GetFirefoxButton`, you would just not include it :/ 
But maybe I am over thinking this and/or not accounting for something that you're thinking of tho.

Reply:
No, that's a good point. If all you're going to display is a GetFirefoxButton, and you know you don't want one, you can just not render one. I'll change this.

Reply:
Actually, I take it back. :-)

I still don't want to mix logic that deals with button types into `InstallButtonWrapper`. I think it's cleaner to have `InstallButtonWrapper` always render a `GetFirefoxButton` and let `GetFirefoxButton` be the one to decide what to do based on the `buttonType`. That seems more encapsulated to me. What do you think?

Reply:
ok fair enough : )